sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -223,6 +223,8 @@ class Analyzer(
           case other => Alias(other, other.toString)()
         }
 
+        val attributeMap = groupByAliases.map(a => (a -> a.toAttribute.withNullability(true))).toMap
```

Comment:
Maybe we need to check the bitmasks? I mean if we something like `GROUPING SETS ( (a,b), a)`, we do not need to change the nullability of `a`, right?